.github/ISSUE_TEMPLATE/team-update.md

```diff
@@ -1,21 +1,47 @@
 ---
-name: Team updates
-about: Giving a team update
-title: "Tech Team Update: [DATE]"
-labels: team-process
+name: "🙌 Team sync"
+about: Syncronize the team's goals and actions
+labels: team-sync
+title: "Team Sync - {{ date | date('dddd, MMMM Do') }}"
 ---
 
-Hey @2i2c-org/tech-team - time to fill in some updates about what you've been up to the last couple of weeks!
-Can folks fill out the [HackMD](https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw) with their own updates? ✨ 
+This is a @2i2c-org/tech-team sync 🎉🎉🎉! This is a way for 2i2c Team Members to triage work, point out issues that require attention, and make sure we are working well as a team. This issue will be closed in 24 hours.
 
-- **Updates HackMD**: https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw
-- **Team Sync history**: https://2i2c.org/team-compass/team/tech/sync/
+### Team Goals
 
-# ToDo
+See the [Team Goal label](https://github.com/2i2c-org/pilot-hubs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Agoal) for a list of team goals that we are currently working towards.
 
-- [ ] Clean up the [HackMD](https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw) for this update
-- [ ] Ping the team members in [`#tech-updates`](https://2i2c.slack.com/archives/C01GLCC1VCN)
-- [ ] Wait 2-3 days
-- [ ] Copy/paste into the `team-compass` repository
-- [ ] Clean up the HackMD
-- [ ] Link to new updates in `team-compass/` in [`#tech-updates`](https://2i2c.slack.com/archives/C01GLCC1VCN)
+### Issues Requiring attention
+
+These issues require attention quickly:
+
+- **New Hubs**: The [Needs Hub issues](https://github.com/2i2c-org/pilot-hubs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Needs+Hub%22) are new hubs that need deployments.
+- **Needs Triage**: The [Needs Triage issues](https://github.com/2i2c-org/pilot/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aneeds%3Atriage) require an initial assessment and labeling.
+- **Priority Issues**: The [Priority Label issues](https://github.com/2i2c-org/pilot-hubs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Apriority) are important and should be given attention over other issues.
+
```

Comment:
Hi everyone! @choldgraf shared this PR with me and I am audacious (I am using the proper word??) enough to make some comments knowing just a little bit about this whole process :wink:

@choldgraf, I think you could be more date-specific in those links??
AFAIK, GitHub searching machinery is powerful enough to filter by dates.
If you can build these links to filter by date, then you will have a "permanent" record of the things happening on some specific week. Otherwise, if I am understanding this correctly, those links are always "live". And if you check the issue in some far away (or not that far away) future, those links (actually the content) will not be relevant for the specific anymore, I think...

Reply:
ah that's clever! Lemme see if we can programmatically do that via the github action template

Reply:
hmmmm now that I think about it, this might not be possible because GitHub attaches metadata to *issue events*, whereas what you're talking about is metadata associated with *when the search is being made*. AKA we might have 5 issues with `priority` open this week, but they may have all been *last updated* last week, and so we can make a query that says "give me a list of priority issues that were last updated between X and Y time" but if we were doing that for *this week*, the returned list would be empty. Does that make sense? Maybe I am missing something about the search API?

Reply:
I think you are right!
I guess the issue here is how to get the picture for each week.
Maybe some GH action to pull that "picture" using the GitHub API?
Maybe is as simple as adding a point here to remember to copy over that data every week?
(I prefer the automation, but maybe I am overthinking this...)

Reply:
We could certainly add something like a "closed this week" update, something like that.

oooh you just gave me an idea for a little hacking project. I wanted an excuse to learn more about https://ghapi.fast.ai/ and I wonder if we could use that to write a github action that:

1. Pulls all open issues w/ `priorty` label
2. Converts it into markdown
3. Posts the GitHub issue using that markdown content to list issues explicitly.

Actually, we could get even fancier in the future and build something like: https://predictablynoisy.com/jupyter-activity-snapshot/jupyter.html

but that is probably not a blocker for this PR 😅